change/@fluentui-react-avatar-e5268f0b-f710-45e9-b1e6-790eb2806807.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Update BadgeProps for consistency",
```

Comment:
This message gets added straight to the changelog so it should be clearer. Something like `Use updated sizes in BadgeProps`

Reply:
I wasn't sure what to do with a set of property changes since there is more than just size in this.  I'll try to be clearer going forward.

Reply:
yes, but for `Avatar` the only change is to change the badge sizes ? is that correct ?

Reply:
I see what you mean.  Can I edit that json directly and push and updated comment?

Reply:
Yes, you can do that